docs/en/differenze_spid_cie.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: ../common/common_definitions.rst
+
+.. _DifferenzeSPID_CIE:
+
+Differences between SPID e CIE id profiles
+------------------------------------------
+
+This section provides the main differences between SPID and CIE id profiles.
+
+Metadata
+++++++++
+
+In CIE id, the OP and RP metadata include the parameters that enable the ID Token encryption (see section `Metadata OP <MetadataOP>`_ and `Metadata RP <MetadataRP>`_)
+
+SPID does not allow the ID Token encryption, thus such a parameters are not required. 
+
+Moreover, CIE id requires the parameter *revocation_endpoint_auth_methods_supported* to be included in the OP metadata, while SPID does not. 
+
+Authorization Endpoint
+++++++++++++++++++++++
+
+SPID profile requires the parameters *client_id* and *response_type* in the HTTP request as mandatory, while in CIE id they are recommended. 
+
+Moreover, in CIE id the response contains the *iss* parameter to mitigate mix-up attacks `I-D.ietf-OAuth-Security-BCP`_. SPID does not require it. 
+
+
+Parameters Scope and Claims
++++++++++++++++++++++++++++
+
+CIE id profile allows an RP to request user attributes using both the *claims* parameter and the *scope* parameter enabling *profile* and *email* values. 
+
+SPID does not allow *profile* and *email* values in the *scope* parameter.
+
+For further details see section :ref:`Parameters Scope and claims <parametri_scope_claims>`.
+
+
+ID Token
+++++++++
+
+In SPID profile the user attributes are not available in the ID Token. 
+
+In CIE id they are available in the ID Token and at the UserInfo endpoint. 
+Moreover, CIE id allows the ID Token encryption.
+
+
+Refresh Token
++++++++++++++
+
+SPID profile allows the Refresh Token to enable the revocable long sessions as defined in `LL.GG. OpenID Connect in SPID <https://www.agid.gov.it/sites/default/files/repository_files/linee_guida_openid_connect_in_spid.pdf>`_ e nell' `Avviso n.41 <https://www.agid.gov.it/sites/default/files/repository_files/spid-avviso-n41-integrazione_ll.gg_._openid_connect_in_spid.pdf>`_ . Whith a Refresh Token an RP can obtain an ID Token which only contains the value *https://www.spid.gov.it/SpidL1* in the *acr* parameter. 
+
+In CIE id the Refresh Token does not allow to obtain an ID Token. Thus, an RP can not obtain a new user authentication with the OP or to renew a pre-existing session. It can be used as a mechanism to obtain from the UserInfo endpoint only the same set of user attributes requested at the initial authentication phase and for which the user has given explicit consent.
```

Comment:
or to renew a pre-existing session -> or renewing a pre-existing one.

Reply:
It can be used as a mechanism to obtain -> In CIE id the UserInfo response contains the same set of user ...